### DISCUSSION TITLE
I got the password but is not working? how to debug it?

### DISCUSSION THREAD

**Discussion by kocoman2** [GerryDazoo/Slinger#481] [General]
I got the password but is not working? how to debug it? 

md: 0x67 err: 0x2b 8
8080 Error Starting Session. Check your admin password in config.ini file!
8080 ERROR: Slingbox session startup failed.
8080 Logging Out

**Comment by lepmail** [comment]
Error 0x2b indicates that your password is correct, but streaming is disabled -- that is, your box is "bricked".  Depending on the model you have you might be able to reset it and get it working again. See other posts. Best of luck to you!

**Reply by socelillyhammer** [reply]
if you have an SB500 here is a suggestion  otherwise, yes follow what @lepmail says

https://www.youtube.com/watch?v=EUyQfpzHRkg

**Reply by GerryDazoo** [reply]
What type of Slingbox are you using? There is no way to unbrick a Solo or Pro.
